site/releases.md

```diff
@@ -64,6 +64,10 @@ Client Guide | API docs
 
 ## News
 
+### [date] Release {{ site.latest_version }} available
```

Comment:
need to update this.

Reply:
Yes, as usual the day of the release.
Thank you for the reminder